var/spack/repos/builtin/packages/psipred/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Psipred(MakefilePackage):
+    """PSIPRED is a simple and accurate secondary structure prediction method, incorporating
+    two feed-forward neural networks which perform an analysis on output obtained from
+    PSI-BLAST (Position Specific Iterated - BLAST). Using a very stringent cross validation
+    method to evaluate the methods performance"""
+
+    homepage = "http://bioinf.cs.ucl.ac.uk/psipred/"
+    url = "http://bioinfadmin.cs.ucl.ac.uk/downloads/psipred/psipred.4.02.tar.gz"
```

Comment:
(question) Out of curiosity, does this have updates not in `https://github.com/psipred/psipred`?

Reply:
Yeah I compared the two. The download from their site is a (marginally) more recent version - and is obviously their 'official' method.